go/billing/api.py

```diff
@@ -228,34 +244,49 @@ def create_transaction_interaction(self, cursor, account_number,
         session_credits = MessageCost.calculate_session_credit_cost(
             session_cost, markup_percent)
 
+        session_unit_credits = MessageCost.calculate_session_unit_credit_cost(
+            session_unit_cost, markup_percent)
```

Comment:
A, just realised I might've misunderstood your comment. There is an associated transaction cost field, `session_unit_cost`. Is that maybe what you meant?